src/flb_log_event_encoder_primitives.c

```diff
@@ -22,6 +22,8 @@
 #include <fluent-bit/flb_byteswap.h>
 #include <stdarg.h>
 
+#define FLB_UINT32_TO_NETWORK_BYTE_ORDER(value) FLB_UINT32_TO_HOST_BYTE_ORDER(value)
```

Comment:
Please move this to `flb_byteswap.h` right after the definition of `FLB_UINT32_TO_HOST_BYTE_ORDER`.

Reply:
Done.